File: backend-java/src/main/java/com/backend/java/backendjava/exception/ExceptionDetails.java
package com.backend.java.backendjava.exception;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ExceptionDetails {
        @JsonFormat(pattern = "yyyy-MM-dd HH:ss")
        private LocalDateTime localDateTime;
        private Integer status;
        private String message;
        private String path;
}

File: backend-kotlin/src/main/kotlin/com/backend/kotlin/backendkotlin/exceptions/ExceptionDetails.kt
package com.backend.kotlin.backendkotlin.exceptions

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime

data class ExceptionDetails(
    @JsonFormat(pattern = "yyyy-MM-dd HH:ss")
    val localDateTime: LocalDateTime,
    val status: Int,
    val message: String,
    val path: String
)
